### PR TITLE
doc: Update AUTHORS file

### PR DESCRIPTION
**Pull request by tgraf**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -134,20 +172,37 @@ Steven Normore                          snormore@digitalocean.com
...
 Umesh Keerthy B S                       umesh.freelance@gmail.com
 Valas Valancius                         valas@google.com
 Vlad Gorodetsky                         v@gor.io
-Vladimir Nachev                         vladimir.penkov.nachev@gmail.com
 Vlad Ungureanu                          vladu@palantir.com
+Vladimir Nachev                         vladimir.penkov.nachev@gmail.com
+Wang Li                                 wangli09@kuaishou.com
+Wayne Haber                             41373231+whaber@users.noreply.github.com
+Weil0ng                                 cuiwl@google.com
+Weilong Cui                             cuiwl@google.com
```
Comment (joestringer) [inline AUTHORS]: Looks like we need to extend `.mailmap`.

File: AUTHORS [hunk 2]
```diff
@@ -3,41 +3,50 @@ off on commits in the Cilium repository:
...
 Amre Shakimov                           amre@covalent.io
-André Martins                           andre@cilium.io
 Andrew Sy Kim                           kim.andrewsy@gmail.com
 Andrey Voronkov                         voronkovaa@gmail.com
+André Martins                           andre@cilium.io
 Anit Gandhi                             anitgandhi@gmail.com
 Archana Shinde                          archana.m.shinde@intel.com
 Arika Chen                              eaglesora@gmail.com
 Arthur Evstifeev                        aevstifeev@gitlab.com
+ArthurChiao                             arthurchiao@hotmail.com
```
Comment (joestringer) [inline AUTHORS]:
```suggestion
Arthur Chiao                             arthurchiao@hotmail.com
```

File: AUTHORS [hunk 3]
```diff
@@ -3,41 +3,50 @@ off on commits in the Cilium repository:
...
 Boran Car                               boran.car@gmail.com
 Brian Topping                           brian@coglative.com
 Bruno Miguel Custódio                   brunomcustodio@gmail.com
 Charles-Henri Guérin                    charles-henri.guerin@zenika.com
+Chris Tarazi                            chris@isovalent.com
 Christine Chen                          christine.chen@datadoghq.com
 Christopher Biscardi                    chris@christopherbiscardi.com
 Connor Jones                            cj@cjmakes.com
 Craig Box                               craig.box@gmail.com
+CybrPunk                                seanmwinn@hotmail.com
```
Comment (joestringer) [inline AUTHORS]: One more for `.mailmap`

File: AUTHORS [hunk 4]
```diff
@@ -50,31 +59,45 @@ Eric Bailey                             e.bailey@sportradar.com
...
 Joe Stringer                            joe@cilium.io
 Johannes Liebermann                     johanan.liebermann@gmail.com
 John Fastabend                          john.fastabend@gmail.com
+John Gardiner Myers                     jgmyers@proofpoint.com
+John Watson                             johnw@planetscale.com
+John Zheng                              johnzhengaz@gmail.com
 Jomen Xiao                              jomenxiao@gmail.com
-Jonathan Davies                         jpds@protonmail.com
 Jon Davies                              jpds@protonmail.com
+Jonathan Davies                         jpds@protonmail.com
```
Comment (joestringer) [inline AUTHORS:94]: One more for `.mailmap`

File: AUTHORS [hunk 5]
```diff
@@ -50,31 +59,45 @@ Eric Bailey                             e.bailey@sportradar.com
...
+Joseph-Irving                           joseph.irving500@gmail.com
+Joshua                                  joshroppo@gmail.com
+Joshua Roppo                            joshroppo@gmail.com
```
Comment (joestringer) [inline AUTHORS]: One more for `.mailmap`

File: AUTHORS [hunk 6]
```diff
@@ -134,20 +172,37 @@ Steven Normore                          snormore@digitalocean.com
...
+Tam Mach                                sayboras@yahoo.com
 Tasdik Rahman                           prodicus@outlook.com
 Thomas Bachman                          tbachman@yahoo.com
 Thomas Graf                             thomas@cilium.io
 Timo Reimann                            ttr314@googlemail.com
 Tobias Klauser                          tklauser@distanz.ch
+Tom Hadlaw                              thomas.hadlaw@hootsuite.com
 Tony Lambiris                           tony@criticalstack.com
+Tony Lu                                 tonylu@linux.alibaba.com
+Tony Lu                                 tonylu@linux.alibaba.com
```
Comment (joestringer) [inline AUTHORS]: doubled up here

**Review by joestringer — CHANGES_REQUESTED** [review]
~I suspect we've resolved several of these in Cilium 1.7 branch, maybe we need to forward-port the `.mailmap` file and amend it.~

(I checked; we haven't. Must just be the new authors this round.)

**Review by aanm — APPROVED** [review]

**Pull request merged**